app/javascript/packs/check-support-ft-unlock.ts

```diff
@@ -0,0 +1,19 @@
+import { isWebauthnPlatformAuthenticatorAvailable } from '@18f/identity-webauthn';
+
+async function checkSupportFtUnlock() {
```

Comment:
Partly the avoid the abbreviation, but also partly to align with other naming, maybe we should call this something along the lines of "platform authenticator available" instead of "support ft unlock"?